x-pack/plugins/security_solution/server/lib/detection_engine/signals/threat_mapping/create_threat_signals.ts

```diff
@@ -90,12 +91,45 @@ export const createThreatSignals = async ({
     perPage,
   });
 
+  const getMatchedThreats: GetMatchedThreats = async (ids) => {
+    const matchedThreatsFilter = {
+      query: {
+        bool: {
+          filter: {
+            ids: { values: ids },
+          },
+        },
+      },
+    };
+    const threatResponse = await getThreatList({
+      callCluster: services.callCluster,
+      exceptionItems: exceptionItems ?? [],
+      threatFilters: [...(threatFilters ?? []), matchedThreatsFilter],
+      query: threatQuery,
+      language: threatLanguage,
+      index: threatIndex,
+      listClient,
+      searchAfter: undefined,
+      sortField: undefined,
+      sortOrder: undefined,
+      logger,
+      buildRuleMessage,
+      perPage: undefined,
+    });
+
+    return threatResponse.hits.hits ?? [];
```

Comment:
I think we can just do:

```ts
return threatResponse.hits.hits
```

Unless the types are wrong here. It looks like hits should always be defined.